src/frontend/Page/Docs.elm

```diff
@@ -561,24 +655,194 @@ viewValueItem { author, project, version } moduleName ownerName valueName =
     url =
       Href.toModule author project version moduleName (Just ownerName)
   in
-  li [ class "pkg-nav-value" ] [ navLink valueName url False ]
+  li [ class "pkg-nav-value" ] [ navModuleLink valueName url False ]
+
+
+
+-- VIEW ABOUT
+
+
+viewAbout : Status Project.PackageInfo -> Status Time.Posix -> Html msg
+viewAbout manifestStatus time =
+  case manifestStatus of
+    Success manifest ->
+      div [ class "block-list pkg-about" ]
+        [ h1 [ class "block-list-title" ] [ text "About" ]
+        , text manifest.summary
+        , viewInstall manifest
+        , viewRelease manifest time
+        , viewLicense manifest
+        , viewDependencies manifest
+        ]
+
+    Loading ->
+      div [ class "block-list pkg-about" ] [ text "" ] -- TODO
+
+    Failure ->
+      div
+        (class "block-list pkg-about" :: Problem.styles)
+        (Problem.offline "elm.json")
+
+
+viewInstall : Project.PackageInfo -> Html msg
+viewInstall manifest =
+  case V.fromTuple (0,19,0) of
```

Comment:
This is kind of annoying, and would be again for 0.19 warnings.
Maybe in the future `Version.fromTuple` could clamp minimal versions to 0 for negative numbers instead of returning a Maybe?